### DISCUSSION TITLE
Urgent request: leave the meeting time alone when I change the day

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#652] [Q&A] [answered]
This has bitten me in the backside more than once and I would like to enter an urgent change request:

Whenever I change the day of a meeting, the times jumps to 12am. This makes no sense to me whatsoever.
Sometimes a group changes their meeting day, but the time usually stays the same. Sometimes the day is given to me in error so I want to be quick and adjust it and then I often miss the time change.

I would like the time, if there is one entered already, to stay the same, even when I change the day. It is very easy to miss and I don't see the reason why it would be changed if all I do is adjust the day....

Pretty please?
Nicky

**Comment by kiyote33** [comment, marked as answer]
Hi Nicky,
I tend to agree with you. I can't think of a reasonable explanation why it would be necessary to change the meeting time when altering the meeting day. It may be that no one ever brought it up before. We will discuss and if the team agrees with us, then this discussion topic will get transferred to the issues list for action.  Thanks for bringing it up...

**Comment by OSAwebsite** [comment]
Sounds good to me! Thanks for letting me know...


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*
